### ISSUE TITLE
Is there a java version of this ?

### ISSUE THREAD

**Issue by csalgo70** [Azure-Samples/azure-cosmosdb-graph-bulkexecutor-dotnet-getting-started#6]
Is there a Java version of the same graph bulkexecutor or its only available on .NET

**Comment by jayanta-mondal** [comment]
@csalgo70: There is no Java version, But we are planning to release a .NET Core version. Let us know if you want a preview.

**Comment by csalgo70** [comment]
Yes .NET core version would bee useful too for our next version, so I can
try azure functions. Please share the preview link.

Thanks,
Kannan

**Comment by jffaust** [comment]
@jayantacs Was the preview for dotnet core ever released?

**Comment by AlouiAmine** [comment]
hello, is there a python version for the bulk executor!?
